website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -229,6 +229,21 @@ Configure [VCS providers](/terraform/cloud-docs/vcs) to use in the organization.
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
+
+### Destruction and Deletion
+
+#### Data Retention Policies
+
+<EnterpriseAlert>
+Data retention policies are exclusive to Terraform Enterprise, and not available in Terraform Cloud. <a href="https://developer.hashicorp.com/terraform/enterprise">Learn more about Terraform Enterprise</a>.
+</EnterpriseAlert>
+
+By default, the **Do not auto-delete data** option is enabled for an organization. This option directs Terraform Enterprise to retain data associated with configuration and state versions, but organization owners can define configurable data retention policies that allow Terraform to _soft delete_ the backing data associated with configuration versions and state versions. Soft deleting refers to marking a data object for garbage collection so that Terraform can delete the object after a set number of days.
+
+Once an object is soft deleted, any attempts to read the object will fail. Until the garbage collection process begins, you can restore soft deleted objects using the APIs described in the [configuration version documentation](/terraform/enterprise/api-docs/configuration-versions) and the [state version documentation](/terraform/enterprise/api-docs/state-versions). Terraform permanently deletes the archivist storage after the garbage collection grace period elapses.
```

Comment:
**Quick Question (same as above):** Should these links point to the cloud docs rather than the enterprise docs, or does it still make sense to point them to `/enterprise` since this feature is enterprise-specific?